### PR TITLE
COMMERCE-4713 Editing a SKU price and going to SKU's price list cause an error

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-price-list-web/src/main/resources/META-INF/resources/edit_instance_tier_price_entry.jsp [hunk 1]
```diff
@@ -32,16 +32,9 @@ CommercePriceList commercePriceList = commercePriceEntry.getCommercePriceList();
...
 
-if ((commercePriceEntry != null) && (commerceTierPriceEntry != null)) {
-	if (commerceTierPriceEntry.getPrice() != null) {
-		price = commerceCurrency.round(commerceTierPriceEntry.getPrice());
-	}
-
-	if (commerceTierPriceEntry.getPromoPrice() != null) {
-		promoPrice = commerceCurrency.round(commerceTierPriceEntry.getPromoPrice());
-	}
+if ((commercePriceEntry != null) && (commerceTierPriceEntry != null) && (commerceTierPriceEntry.getPrice() != null)) {
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-price-list-web/src/main/resources/META-INF/resources/edit_instance_tier_price_entry.jsp:36]: @riccardo-alberti is commercePriceEntry != null  condiction needed?

Comment (riccardo-alberti) [reply modules/apps/commerce/commerce-price-list-web/src/main/resources/META-INF/resources/edit_instance_tier_price_entry.jsp]: @marco-leo i removed it as the price is taken from tierPriceEntry

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**